widget/splitcontainer.go

```diff
@@ -0,0 +1,263 @@
+package widget
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/theme"
+)
+
+var (
+	dividerThickness = theme.Padding() * 2
+	dividerLength    = theme.Padding() * 6
+	handleThickness  = theme.Padding() / 2
+	handleLength     = theme.Padding() * 4
```

Comment:
Indeed great work!
Saw minor issue here. These values will be frozen at startup and won't react on future theme change. It will be better to declare constants with ratio (x100) and then calculate actual values at renderer.

Reply:
Nice catch @pymq!